docs/installationguide/debian.rst

```diff
@@ -3,6 +3,92 @@ Debian
 
 .. contents::
 
+Easy Way: Debian 11 (Bullseye)
+------------------------
+
+This procedure will guide you through the installation of ZoneMinder on Debian 11 (Bullseye).
+
+**Step 1:** Setup Sudo (optional but recommended)
+
+By default Debian does not come with sudo, so you have to install it and configure it manually.
+This step is optional but recommended and the following instructions assume that you have setup sudo.
+If you prefer to setup ZoneMinder as root, do it at your own risk and adapt the following instructions accordingly.
+
+::
+
+    apt install sudo
+    usermod -a -G sudo <username>
+    exit
+
+Now your terminal session is back under your normal user. You can check that 
+you are now part of the sudo group with the command ``groups``, "sudo" should
+appear in the list. If not, run ``newgrp sudo`` and check again with ``groups``.
+
+**Step 2:** Update system and install zoneminder
+
+Run the following commands.
+
+::
+
+    sudo apt update
+    sudo apt upgrade
+    sudo apt install zoneminder
+
+**Step 3:** Initialize database
+
+Run the following commands.
+
+::
+
+    cat /usr/share/zoneminder/db/zm_create.sql | sudo mariadb
+    echo 'grant lock tables,alter,create,select,insert,update,delete,index on zm.* to 'zmuser'@localhost identified by "zmpass";' | sudo mariadb
+
+**Step 4:** Setup permissions for zm.conf
+
+To make sure zoneminder can read the configuration file, run the following command.
+
+::
+
+    sudo chgrp -c www-data /etc/zm/zm.conf
+
+**Step 5:** Edit Timezone in PHP
```

Comment:
This section is no longer required.  You set the timezone in options now.

Reply:
Great! I removed this in a new commit.